GiniVision/Classes/Core/GiniScreenAPICoordinator+Camera.swift

```diff
@@ -134,10 +144,10 @@ extension GiniScreenAPICoordinator: CameraViewControllerDelegate {
     
     private func showMultipageNewFeatureDialog() {
         let alertDialog = AlertDialogController(giniConfiguration: giniConfiguration,
-                                                title: "This is the title",
-                                                message: "This is the subtitle",
+                                                title: "Multi-Page Rechnungsanalyse",
+                                                message: "Jetzt ist es möglich, eine Rechnung mit mehreren Seiten zu analysieren",
                                                 image: UIImageNamedPreferred(named: "multipageIcon"),
-                                                buttonTitle: "Let's scan!",
+                                                buttonTitle: "Zur Kamera",
```

Comment:
All strings here are not localized and/or configurable.

Reply:
This is part of the Dialog that is removed in a future PR as I told you.